modules/candidate_parameters/jsx/FamilyInfo.js

```diff
@@ -103,25 +98,23 @@ var FamilyInfo = React.createClass({
         var link = "?candID=" + candID + "&identifier=" + candID;
 
         familyMembersHTML.push(
-                  <div>
-                      <StaticElement
-                          label="Family Member DCCID"
-                          text={<a href={link}>{candID}</a>}
-                      />
-                      <StaticElement
-                          label="Relation Type"
-                          text={relationshipOptions[relationship]}
-                      />
-                      <ButtonElement
-                          label="Delete"
-                          type="button"
-                          onUserInput={
-                              this.deleteFamilyMember.bind(null, candID, key, candidateList)
-                          }
-                      />
-                      <hr />
-                  </div>
-              );
+          <div key={key}>
```

Comment:
They key is added to fix react warning/impove performance. It has nothing to do with being able to add empty option. But I can set fields to `required` as a quick fix